monitor/bitbucket.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+const countPerPage = 500
+
+var workspaceMembersURLPath = "/2.0/users/{workspace}/members"
+
+type bitbucketMember struct {
+	DisplayName   string `json:"display_name"`
+	Nickname      string `json:"nickname"`
+	Has2faEnabled *bool  `json:"has_2fa_enabled"`
+}
+
+type bitbucketMembers struct {
+	Values []bitbucketMember `json:"values"`
+	Size   int               `json:"size"`
+}
+
+func (members *bitbucketMembers) getNon2svMembers() []bitbucketMember {
+	var non2svMembers []bitbucketMember
+
+	for _, member := range members.Values {
+		if member.Has2faEnabled == nil {
```

Comment:
is Has2faEnabled ever false? Why use the pointer instead of just a bool?

Reply:
Fixed, Added a check for false values in case they start passing true/false. I've never seen a false value, but doesn't mean it doesn't exist.